### DISCUSSION TITLE
Make a COMMITment 💕 [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#109609] [Pull Requests]
![Header for the Pull Requests category check-in shows a blue starry background with various planets in the distance. The left has some clouds and an octocat in a Hawaiian shirt holding a GitHub mug. To the right there is a blue and red rocket ship taking off.](https://github.com/community/community/assets/92824518/b7f50bb7-5979-4c6f-92aa-8f3dc5347b99)

February is the month of love and what better way to show your love for GitHub by making some commits!  

### ⚡Pull Requests Praise: Thanks to these fine folks for keeping our Community awesome 
* @spenserblack: Is a world class helper all around the community, but I loved [his response](https://github.com/orgs/community/discussions/108528#discussioncomment-8496824) on Pull Request search filters, explained the answer, pulled some context from the docs, and shared his own experiences. Thank you! 
* @Julian-dev28: On the subject of filtering, @Julian-dev28 [asked a great question](https://github.com/orgs/community/discussions/102169) about filtering out team contributions. They’re still looking for suggestions, if you have a solution, be sure to share it. 

### ⬆️ Level up: Resources to take your skills to new heights ☁️
* **Build code security skills with the GitHub Secure Code Game**: Learn to find and fix security issues while having fun with Secure Code Game, we just added new challenges focusing on JavaScript, Python, Go, and GitHub Actions! [Get all the details](https://github.blog/2024-02-15-build-code-security-skills-with-the-github-secure-code-game/). 
* **GitHub Skills - Review Pull Requests**: All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving. [Learn when and how to request a review; how to provide a review of someone else's pull request in this GitHub Skills course.](https://github.com/skills/review-pull-requests). 

### 📣 Announcements: The latest updates in the Community 🆕
* **Achievements are no longer available in the Community**:  In response to this fast growing issue and comments from members around the galaxy brain achievement, we have made the decision to turn off the ability to earn Achievements in our Community. [Read more for details](https://github.com/orgs/community/discussions/106536). 

Have you made any impactful commits to your project recently? What were they?

**Comment by Julian-dev28** [comment]
Lol thanks @queenofcorgis

**Comment by queenofcorgis** [comment]
Commit to reading our newest check-in: [PRs check-in for March](https://github.com/orgs/community/discussions/113573) 🌈
